lib/clAdapters/activeSync/calendar/index.ts

```diff
@@ -102,6 +104,203 @@ export default class ActiveSyncCalendarAdapter extends ActiveSyncBaseAdapter {
     return this;
   }
 
+  expandDaysOfWeek = function(daysOfWeek: number) {
```

Comment:
Nit/non-blocking: Class syntax for prototype methods (e.g. `expandDaysOfWeek(daysOfWeek: number)` vs `= function()`, here an elsewhere.